packages/material/scss/pager/_layout.scss

```diff
@@ -0,0 +1,47 @@
+@import "~@progress/kendo-theme-default/scss/pager/layout";
+
+@include exports ('pager/layout/material') {
+    $pager-nav-padding: 6px !default;
+    $pager-elements-margin: 8px !default;
+    $pager-elements-margin-lg: 32px !default;
+    $pager-page-link-size: 36px !default;
+
+    .k-pager-wrap {
+        padding: $pager-padding-y $pager-padding-x;
+
+        .k-icon {
+            width: $icon-size;
+            height: $icon-size;
+            line-height: $line-height-lg;
+        }
+
+        .k-pager-nav {
+            padding: $pager-nav-padding;
+            width: auto;
+            height: auto;
```

Comment:
Perhaps using $pager-buttons-size from theme-default should remove the need for this rule, as well as the extra variable?

This will also remove the need for the last rule in this file.